lib/serverless-tools/deployer/ecr_pusher.rb

```diff
@@ -11,6 +11,7 @@ def initialize(client:, git:, config:)
 
       def push(local_image_name:)
         system("docker tag #{local_image_name} #{tagged_image_uri}")
+        system("aws ecr get-login-password | docker login --username AWS --password-stdin #{repository_uri}")
```

Comment:
How does this authenticate to AWS, will `deploy push` need to be run under `aws-vault exec $SOME_PROFILE`?